entry.go

```diff
@@ -212,68 +224,68 @@ func (entry Entry) HasCaller() (has bool) {
 
 // This function is not declared with a pointer value because otherwise
```

Comment:
This comment should be updated since it's a pointer method now.

Is it safe to use via pointer?

Reply:
Yes it is safe, the comment should indeed be updated.

Reply:
Just wanted to point out since I was checking back through this that it's safe because of the `entry.Dup()` call in the new version.

Hooks are now safe to modify entry because any log call on entry duplicates the entire entry.